CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ platform does not support wrappers anymore (it was used for testing).
 - Added mypy and ruff to the CI for the files touched by a change granting a 
   higher quality and preservation of the code #2626 #2621
 - Updated base images of micromamba and debian for security update #2610
+- Improving the error message handling #2651
```

Comment:
It'd be better if we explained the issue here. I believe this error was found by Tracy, an operator of DestinE. If they want to confirm this has been fixed, they will search for that in the changelog. But this changelog alone doesn't help them, unless they remember the issue number. Probably better to mention the cryptic part, e.g. "Improve error reporting for incorrect YAML syntax (too cryptic) #...". That way Tracy will immediately recognize their issue.